client/app/services/notifications.js

```diff
@@ -1,8 +1,10 @@
 import debug from 'debug';
 
+export let Notifications = null; // eslint-disable-line import/no-mutable-exports
+
 const logger = debug('redash:notifications');
 
-function Notifications(currentUser, Events) {
+function NotificationsService(currentUser, Events) {
```

Comment:
Because `Events` can be replaced with `recordEvent` this doesn't have to be an Angular service at all. We can import it directly in `view.js` (its only user).